stories/src/helpers.tsx

```diff
@@ -68,7 +69,7 @@ export const colorSchemeKnob = (initial?: string[]) =>
   )
 
 export const tableKnob = (initial?: Table) =>
-  select('Data', {CPU}, initial || CPU)
+  select('Data', {CPU, CPUString}, initial || CPU)
```

Comment:
Since this new knob is intended only for Mosaic Plots, why not create another knob instead of adding to the tableKnob.

Perhaps something like `mosaicDataKnob`?

Reply:
@TCL735 I wonder if it is useful to have a concept of the giraffe plot failing gracefully when it encounters data it doesn't expect? I was hoping to ask @reparker837 and @rbose22 to have the mosaic plot display an error instead of breaking when it finds numeric values, and so I wonder if it could be a goal to have other graphs display a similar error?

Reply:
Sure. That sounds good. We can leave this knob as it is changed, for now.